### PR TITLE
Review controller

### PR DESCRIPTION
**Pull request by sylviabbt**

### CODE REVIEW

**Comment by sylviabbt** [comment]
reviews-controller, again tbc after the views

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      redirect_to booking_path(@review.booking)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def review_params
+    params.require(:review).permit(:content, :rating, :booking_id)
```
Comment (ivyyu10) [inline app/controllers/reviews_controller.rb]: we can also remove booking_id on this but need to create a few more lines above in the create (similar to the bookings controller) let's have a look through together and test!

**Review by ivyyu10 — CHANGES_REQUESTED** [review]
Please add the views pages with controllers so we can test with front end right away before we style!

**Comment by sylviabbt** [comment]
changes in controller done
also removed double nesting from the routes
@ivyyu10

**Review by ivyyu10 — APPROVED** [review]

**Pull request merged**